src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewFieldBuilder.java

```diff
@@ -53,4 +63,23 @@ public CaseViewField build(CaseField caseField, CaseEventField eventField, Objec
             .map(eventField -> build(caseFieldMap.get(eventField.getCaseFieldId()), eventField, data != null ? data.get(eventField.getCaseFieldId()) : null))
             .collect(Collectors.toList());
     }
+
+    private void getFieldTypeWithDynamicListsPopulated(CaseViewField caseField, Object value) {
+
+        if (caseField.getFieldType().getType().equals(DYNAMIC_LIST) && value != null) {
+            caseField.setValue(((ObjectNode) value).get(DEFAULT));
+            caseField.getFieldType().setFixedListItems(processDynamicList((ObjectNode) value));
+        }
+    }
+
+    private List<FixedListItem> processDynamicList(ObjectNode value) {
+        List<FixedListItem> result = new ArrayList<>();
+        value.get(DYNAMIC_LIST_ITEMS).elements().forEachRemaining(dynamicList -> {
+            FixedListItem listItem = new FixedListItem();
```

Comment:
How difficult would be to rename `FixedListItem` to `ListItem`? Don't think we risk breaking anything

Reply:
Should leave we this when we will be standarding fixed-lists to lists across the applications?